packages/host/app/services/command-service.ts

```diff
@@ -162,7 +166,7 @@ export default class CommandService extends Service {
 }
 
 type PatchPayload = { card_id: string } & PatchData;
-type SearchPayload = { card_id: string; filter: CardTypeFilter };
+type SearchPayload = { card_id: string; filter: CardTypeFilter | EqFilter };
```

Comment:
Only made an incremental addition. This filter type should eventually match type `Filter` from `runtime-common/query` module to make sure of all our filtering and sorting capabilities.